src/main/java/ch/njol/skript/expressions/ExprElement.java

```diff
@@ -32,23 +32,29 @@
 import ch.njol.util.Kleenean;
 import ch.njol.util.StringUtils;
 import ch.njol.util.coll.CollectionUtils;
+import com.google.common.collect.Iterators;
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
 
 import java.lang.reflect.Array;
+import java.util.Iterator;
 
 @Name("Element of")
 @Description({"The first, last or a random element of a set, e.g. a list variable.",
 		"See also: <a href='#ExprRandom'>random</a>"})
 @Examples("give a random element out of {free items::*} to the player")
-@Since("2.0")
+@Since("2.0, INSERT VERSION (relative to last element)")
 public class ExprElement extends SimpleExpression<Object> {
 
 	static {
-		Skript.registerExpression(ExprElement.class, Object.class, ExpressionType.PROPERTY, "(-1¦[the] first|1¦[the] last|0¦[a] random|2¦%-number%(st|nd|rd|th)) element [out] of %objects%");
+		Skript.registerExpression(ExprElement.class, Object.class, ExpressionType.PROPERTY, "(0:[the] first|1:[the] last|2:[a] random|3:[the] %-number%(st|nd|rd|th)|4:[the] %-number%(st|nd|rd|th) last) element [out] of %objects%");
```

Comment:
Isn't `2nd to last element` correct, instead of `2nd last element`?

Reply:
Doesn't `2nd to last element` means from the 2nd to the last element (a subset list)?
The idea of `2nd last element` is to return the 2nd element from the last of the list (reversed/opposite list)

Reply:
![image](https://user-images.githubusercontent.com/29547183/201528149-c46d5c28-3a3a-455e-9e22-1e890ff1b6b6.png)


If we want to get fancy with it, we could also add `penultimate` to the syntax.

And if we we want to get even fancier with it we could add `antepenultimate`:
![image](https://user-images.githubusercontent.com/29547183/201528180-cf79224d-e78c-40f5-8b9b-ac30531a0e74.png)

Reply:
Ooh, that's some new information added to my English language :) 

In this case I would say add an optional [to] to the syntax

as for your suggestion those are totally new words to me 😂 so it's up to the rest of the team

Reply:
I agree that an optional `to` is probably the best option. what do you think @tpgamesnl?

Reply:
Fine with me